verde/base/base_classes.py

```diff
@@ -267,7 +275,8 @@ def grid(
             3D vector data.
         projection : callable or None
             If not None, then should be a callable object
-            ``projection(easting, northing) -> (proj_easting, proj_northing)``
+            ``projection(longitude, latitude) ->
+            (proj_easting, proj_northing)``
```

Comment:
Not necessarily. The projection could be reversed for a spherical gridder that we want to make a Cartesian grid. That's why it's hard to be too specific in these classes.

Reply:
Agree